### ISSUE TITLE
Full Accessibility & 508 Office Audit - Caregivers

### ISSUE THREAD

**Issue by ala-yna** [department-of-veterans-affairs/va.gov-team#10743]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] Product Manager: create issue and fill in feature name in the title and other bolded information appropriately
- [ ] Practice area reviewers create sub-issues attached to this one with feedback for your team on *feature name*
- [ ] **Requesting team** completes attached feedback tickets based on definition of done included in each practice area feedback ticket (may differ for each practice area, please refer to attached tickets for further instruction)
- [ ] All requesting team attendees complete brief [VSP Collaboration Cycle Feedback](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before) survey

## Artifacts - _please provide the following artifacts when you request a full review_
- Access information for the tool or feature in staging: https://staging.va.gov/family-member-benefits/apply-for-caregiver-assistance-form-10-10cg/ ( + MVI users to trigger successful application)

- Certify that the following required accessibility tests are complete by checking each of these boxes (your accessibility specialist is available to help). Recommended items are optional, but highly encouraged. Please provide links to known issues.

    **Required:**
    - [x] Read the [accessibility staging review documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) 
    - [x] Conduct axe browser scan, and document or address issues found 
    - [x] Ensure end-to-end tests - including the axe check, pass 
    - [x] Check color pairing for proper contrast 
    - [x] Check for colorblindness issues 
    - [x] Zoom to 200% at 1280px wide and confirm readability, no info cut off, etc 
    - [x] Repeat for 300% 
    - [x] Repeat for 400% 
    - [x] Navigate all pages and core functionality with keyboard only 
    - [x] Test all pages and core functionality happy path with MacOs + VoiceOver and/or NVDA+Firefox 
    
    **Recommended:**
    - [x] Read the [accessibility post-launch processes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/post-launch-audit-processes.md) 
    - [ ] Add keyboard end-to-end tests in CI/CD test runs 
    - [x] Conduct axe-coconut browser scan, document or address issues found 
    - [ ] Test with [WAVE tool](https://wave.webaim.org/) 

Known accessibility issues:

* [#9305 508-defect-2 [FOCUS MANAGEMENT, SCREENREADER]: Focus on page load SHOULD be consistent](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9305)

* [#9174 508-defect-3 [SEMANTIC MARKUP]: Radio inputs with the same name attribute value MUST be part of a group](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9174)

* [#8558 508-defect-2 [SCREENREADER, SEMANTIC MARKUP]: Document SHOULD not have more than one contentinfo landmark](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8558)

* [#8556 508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8556)

* [The base 10-10CG PDF is not compliant; VA 508 Office is working with the owner to remediate.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10559#issuecomment-654289404)

* [The Additional Info PDF page for overflow content is not compliant; VSP is investigating a solution.](https://dsva.slack.com/archives/CBU0KDSB1/p1596469767430400)

## Additional Notes
As part of this review, VSP will send your product to VA's 508 office for approval (as applicable)

**Comment by jenstrickland** [comment]
@ala-yna Please let me know when your team has checked all the boxes under the Required accessibility tests, and then I will conduct the full audit and send to the 508 Office. Grazi!

**Comment by LinnJS** [comment]
Looks like we cant test WAVE that I know of since staging and prod are password protected and result in a 401 unauthorized error

![image](https://user-images.githubusercontent.com/23741323/90672333-f9267000-e223-11ea-8006-1e60e9a61d5e.png)

**Comment by jenstrickland** [comment]
You're all good! axe and axe-coconut are fine. 😄

**Comment by ala-yna** [comment]
Hey @jenstrickland ! Just checking - is the Full Accessibility & 508 Office Audit complete? Was that the only finding?

**Comment by jenstrickland** [comment]
I'm sorry, @ala-yna , it's still in-progress. I'm drowning. The new a11y specialist can't arrive, and be on-boarded soon enough. Apologies for the delay! 

On the subject, though, I hear there's a custom form submission error message… Do you have a documented test case that I can use to include it in my 508 audit, please?

**Comment by jenstrickland** [comment]
@1Copenut This audit is complete and ready for your VSP review.

cc @ala-yna for visibility

**Comment by 1Copenut** [comment]
@ala-yna I've reviewed Jennifer's feedback and don't have anything to add. She and I consulted on the `508-severity-1` issue about the email address needing validation. Let me know if you have any questions.

**Comment by jenstrickland** [comment]
@ala-yna @1Copenut Thanks for the review! I'll collect and send to the VA 508 Office this week for their audit.

**Comment by ala-yna** [comment]
Audit is complete - marking as closed!
